plugins/replay/src/main/java/forklift/replay/ReplayES.java

```diff
@@ -77,8 +76,9 @@ public void run() {
         });
 
         this.producer = connector.getQueueProducer(ReplayConsumer.class.getAnnotation(Queue.class).value());
-
-        this.consumer = new Consumer(ReplayConsumer.class, connector,
+        final Forklift forklift = new Forklift();
```

Comment:
Why not reuse the `Forklift` instance from when this plugin was created like `RetryES` does?